addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -89,11 +97,57 @@ class ComposerTextInput extends Component {
     // Private
     //--------------------------------------------------------------------------
 
-        /**
+    async detectDelimiter() {
+        this._mentionWord = this.validateKeyword('@', false);
+        if (this._mentionWord !== false) {
```

Comment:
is false important in this case? otherwise just:
```suggestion
        if (this._mentionWord) {
```

Reply:
False is required as ```_validateKeyword``` can return empty string